main/guides/getting-started/contract-rpc.md

```diff
@@ -101,11 +101,11 @@ $ agoric follow -lF :published.agoricNames.brand
 ]
 ```
 
-::: tip vstorage viewer by p2p
+::: tip: use vstorage viewer
```

Comment:
```suggestion
::: tip VStorage Viewer```

The `:` after tip doesn't render the tip section. Also, this heading makes more sense.